.eslintrc.json

```diff
@@ -24,7 +24,7 @@
     },
     "settings": {
         "react": {
-            "version": "16.4.0"
+            "version": "detect"
```

Comment:
This allows the React ESLint plugin to [detect](https://github.com/yannickcr/eslint-plugin-react#configuration) the appropriate React version automatically.